### DISCUSSION TITLE
absorbing boundary

### DISCUSSION THREAD

**Discussion by zhanxianchanghe** [Anura3D/Anura3D_OpenSource#245] [General]
Hello, I want to use the absorbing boundary to set the viscoelastic boundary, is this operation feasible? How do I determine the parameters alpha and delta of the absorption boundary, whether they are directly input to the spring stiffness k and damping coefficient c?

**Comment by zhanxianchanghe** [comment]
How are 0.8 and 4 in the example calculated?

**Comment by icy1237** [comment]
Hello, have you resolved this?
